pkgs/stdenv/cross/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib
-, system, platform, crossSystem, config
+, localSystem, crossSystem, config
```

Comment:
Ok, sorry for my super crafty names:

The 3 things which packages ought to solely use are `buildPlatform` `hostPlatform` and `targetPlatform`, one for each autoconf platform.

The dichotomy of `localSystem` and `crossSystem` is strictly for purposes of choosing a stdenv, where a more basic "dev" vs "deploy" is adequate for choosing the *goals* of the bootstrap.

Finally,  there is `stdenv.cross` for target (null if ==build) and `crossSystem` (again!) for host (null if ==build). [The two `crossSystem`s are actually equivalent as it turns out, but damn, I'm evil.]

So please, give me better nomenclature :)!